### PR TITLE
Fc update

### PR DESCRIPTION
**Pull request by ProtoPrism**

### CODE REVIEW

File: resources/views/pages/study/representation/faculty-council.blade.php [hunk 1]
```diff
@@ -1,5 +1,4 @@
 @extends('pages.study')
-@section('title', "Faculty Council - T.F.V. 'Professor Francken'")
```
Comment (MarkRedeman) [inline resources/views/pages/study/representation/faculty-council.blade.php]: Why did you remove the title?

File: resources/views/pages/study/representation/faculty-council.blade.php [hunk 2]
```diff
@@ -45,6 +35,22 @@
 		<br>
 		
 	</div>
+	<div class="col">
+		<iframe width="560" height="315" src="https://www.youtube.com/embed/h0yKz8iRgbY" frameborder="0" allowfullscreen></iframe>
+		<br>
```
Comment (MarkRedeman) [inline resources/views/pages/study/representation/faculty-council.blade.php]: Instead of adding all of these `<br>` tags you could add some margin or padding to the `<iframe>`. Bootstrap has some helpers for this, see [spacing](https://v4-alpha.getbootstrap.com/utilities/spacing/).

File: resources/views/pages/study/representation/faculty-council.blade.php (hunk 1 above)
Comment (ProtoPrism) [inline resources/views/pages/study/representation/faculty-council.blade.php]: Copy Paste error, will add again.
